cmd/msi/utils_windows.go

```diff
@@ -34,14 +38,10 @@ func installStateToString(state gowin32.InstallState) string {
 	return "<Unsupported>"
 }
 
-var msiInitialized = false
+var msiInitialized sync.Once
 
 func initMsi() {
-	if msiInitialized {
-		return
-	}
-
-	gowin32.SetInstallerInternalUI(gowin32.InstallUILevelNone)
-
-	msiInitialized = true
+	msiInitialized.Do(func() {
+		gowin32.SetInstallerInternalUI(gowin32.InstallUILevelNone)
+	})
```

Comment:
This change stands out from the rest (it belongs in a separate PR)

sync.Once does fix one issue - goroutines being created in quick succession all ending up in initMsi() - several of them might call `gowin32.SetInstallerInternalUI` because they'd read `msiInitialized` as false.

Not only would that be very rare, I'm not sure how the windows installer API would behave when called for multiple threads. I'd be interested in a test case that shows the issue, giving us a clear rationale for "fixing" it.